binaryaudit/dnf.py

```diff
@@ -21,33 +23,36 @@ def process_downloads(source_dir, new_json_file, old_json_file, output_dir, buil
             build_id (str): The build id
             product_id (str): The product id
             db_conn: The db connection
-
+            remianing_files (int): The number of files left after filtering
         Returns:
             overall_status (str): Returns "fail" if an incompatibility is found in at least 1 RPM, otherwise returns "pass"
     '''
     try:
-        overall_status = "pass"
+        processed_files = 0
+        overall_status = "PASSED"
         conf_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), "../conf")
         if not os.path.exists(source_dir + "old"):
             os.mkdir(source_dir + "old")
         old_rpm_dict = {}
         with open(new_json_file, "r") as file:
             data = json.load(file)
         for key, values in data.items():
+            processed_files += len(data[key])
             for value in values:
                 with rpmfile.open(source_dir + value) as rpm:
                     name = rpm.headers.get("name")
                 old_rpm_name = download(key, source_dir, name, old_rpm_dict)
             if old_rpm_name == "":
+                util.note("Processed " + str(processed_files) + " of " + str(remaining_files) + " files")
                 continue
             with open(old_json_file, "w") as outputFile:
                 json.dump(old_rpm_dict, outputFile, indent=2)
             ret_status = generate_abidiffs(key, source_dir, new_json_file, old_json_file, output_dir,
                                            conf_dir, build_id, product_id, db_conn, cleanup)
-            util.note("Status: " + ret_status)
-
-            if ret_status == "fail":
-                overall_status = "fail"
+            util.note("Status: " + str(ret_status))
```

Comment:
Here and at other places, could use the `format` method instead. Eq `"Status: {}".format(ret_status)`. Perhaps would make sense to have a separate PR on switching to use `.format`, it seems to be the standard way to do this in Python. Please see here:
 https://docs.python.org/3/library/string.html#format-string-syntax

Thanks